docs/topics/commonly_used_plugins.rst

```diff
@@ -10,284 +10,65 @@ Some commonly-used plugins
-<http://github.com/FlashJunior/OSFlashVideoPlayer>`_. When uploading videos use either
-``.flv`` files or H264 encoded video files.
-
-Please install it using ``pip`` or similar and be sure you have the following in your project's
-:setting:`django:INSTALLED_APPS` setting::
-
-    INSTALLED_APPS = (
-        # ...
-        'djangocms_video',
-        # ...
-    )
-
-There are some settings you can set in your ``settings.py`` to overwrite some
-default behaviour:
-
-* ``VIDEO_AUTOPLAY`` ((default: ``False``)
-* ``VIDEO_AUTOHIDE`` (default: ``False``)
-* ``VIDEO_FULLSCREEN`` (default: ``True``)
-* ``VIDEO_LOOP`` (default: ``False``)
-* ``VIDEO_AUTOPLAY`` (default: ``False``)
-* ``VIDEO_BG_COLOR`` (default: ``"000000"``)
-* ``VIDEO_TEXT_COLOR`` (default: ``"FFFFFF"``)
-* ``VIDEO_SEEKBAR_COLOR`` (default: ``"13ABEC"``)
-* ``VIDEO_SEEKBARBG_COLOR`` (default: ``"333333"``)
-* ``VIDEO_LOADINGBAR_COLOR`` (default: ``"828282"``)
-* ``VIDEO_BUTTON_OUT_COLOR`` (default: ``"333333"``)
-* ``VIDEO_BUTTON_OVER_COLOR`` (default: ``"000000"``)
-* ``VIDEO_BUTTON_HIGHLIGHT_COLOR`` (default: ``"FFFFFF"``)
-
-You should take care that the directory defined by the configuration setting
-:setting:`CMS_PAGE_MEDIA_PATH` (by default ``cms_page_media/`` relative to
-:setting:`django:MEDIA_ROOT`) is writeable by the user under which django will be
-running.
-
-.. note:: For more advanced use cases where you would like to upload your media
-          to a central location, consider using  `django-filer`_ with
-          `django filer CMS plugin`_ and its ``cmsplugin_filer_video`` component
-          instead.
-
-.. _django-filer: https://github.com/stefanfoulis/django-filer
-.. _django filer CMS plugin: https://github.com/stefanfoulis/cmsplugin-filer
-
-..  module:: djangocms_twitter
-
-..  class:: djangocms_twitter.cms_plugins.TwitterRecentEntriesPlugin
-
-..  class:: djangocms_twitter.cms_plugins.TwitterSearchPlugin
-
-*******
-Twitter
-*******
-
-We recommend one of the following plugins:
-
-* https://github.com/nephila/djangocms_twitter
-* https://github.com/changer/cmsplugin-twitter
-
-.. warning:: These plugins are not currently compatible with Django 1.7.
-
-..  module:: djangocms_inherit
-
-..  class:: djangocms_inherit.cms_plugins.InheritPagePlaceholderPlugin
-
-*******
-Inherit
-*******
+These packages are no longer officially guaranteed support by the django CMS project, but they have good community support.
 
-Available on `GitHub (divio/djangocms-inherit) <http://github.com/divio/djangocms-inherit>`_
-and on `PyPi (djangocms-inherit) <https://pypi.python.org/pypi/djangocms-inherit>`_.
+*	`django CMS Inherit <https://github.com/divio/djangocms-inherit>`_ - renders the plugins from a specified page (and language) in its place
+*	`django cms Column <https://github.com/divio/djangocms-column>`_ - layout page content in columns
+*	`GoogleMap <http://github.com/divio/djangocms-googlemap>`_ - displays a map of an address on your page. Both address and
```

Comment:
@evildmp oh that's then missing. It is a core addon, you can also check the wiki for the current updated list.